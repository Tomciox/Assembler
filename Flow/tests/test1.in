5 3 0.5
1.0 2.0 3.0 4.0 5.0
6.0 7.0 8.0 9.0 10.0
11.0 12.0 13.0 14.0 15.0
3
1.0 2.0 3.0
10.0 20.0 30.0
100.0 200.0 300.0
