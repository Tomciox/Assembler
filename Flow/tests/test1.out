4.000000 6.000000 7.000000 8.000000 9.000000 
0.000000 5.500000 6.500000 7.500000 8.500000 
0.000000 6.000000 7.000000 8.000000 9.000000 

13.000000 1.750000 5.500000 6.500000 7.500000 
32.000000 -0.250000 6.000000 7.000000 8.000000 
25.000000 -0.250000 5.500000 6.500000 7.500000 

159.500000 21.500000 1.000000 6.000000 7.000000 
271.000000 36.125000 -2.875000 4.500000 5.500000 
253.500000 28.500000 0.000000 6.000000 7.000000 

